Sources/StreamChatUI/ChatMessageList/ChatVC.swift

```diff
@@ -146,14 +146,14 @@ open class _ChatVC<ExtraData: ExtraDataTypes>: ViewController,
 
         let subtitle = UILabel()
         subtitle.textAlignment = .center
-        subtitle.font = uiConfig.font.subheadline
+        subtitle.font = uiConfig.font.footnote
         subtitle.textColor = uiConfig.colorPalette.subtitleText
 
         let titleView = UIStackView(arrangedSubviews: [title, subtitle])
         titleView.axis = .vertical
-        titleView.spacing = 2
 
         navigationItem.titleView = titleView
+        navigationController?.navigationBar.topItem?.title = ""
```

Comment:
Why is this needed? This doesn't look exactly safe to modify someone else's navigation item :)